widgets/blueprintCatalog/src/RepositoryTable.tsx

```diff
@@ -34,49 +43,47 @@ const RepositoryTable: FunctionComponent<RepositoryViewProps> = ({
-                show={fieldsToShow.includes(t('configuration.fieldsToShow.items.updated'))}
-            />
+            <DataTable.Column label={t('name')} width="25%" show={fieldsToShow.includes(t('name'))} />
+            <DataTable.Column label={t('description')} width="40%" show={fieldsToShow.includes(t('description'))} />
+            <DataTable.Column label={t('created')} width="12%" show={fieldsToShow.includes(t('created'))} />
+            <DataTable.Column label={t('updated')} width="12%" show={fieldsToShow.includes(t('updated'))} />
             <DataTable.Column width="11%" />
 
             {data.items.map(item => {
-                const isReadmeLoading = readmeLoading === item.name;
-                const isBlueprintUploaded = data.uploadedBlueprints.includes(item.name);
+                /* eslint-disable camelcase */
+                const {
+                    id,
+                    name,
+                    description,
+                    created_at,
+                    updated_at,
+                    image_url,
+                    isSelected,
+                    html_url,
+                    readme_url,
+                    zip_url,
+                    main_blueprint
+                } = item;
+                const isReadmeLoading = readmeLoading === name;
+                const isBlueprintUploaded = data.uploadedBlueprints.includes(name);
 
                 return (
                     <DataTable.Row
-                        key={item.id}
-                        className={`bp_${item.name}`}
-                        selected={item.isSelected}
+                        key={id}
+                        className={`bp_${name}`}
+                        selected={isSelected}
                         onClick={() => onSelect(item)}
                     >
                         <DataTable.Data>
-                            <ExternalBlueprintImage url={item.image_url} width={30} />{' '}
-                            <a href={item.html_url} target="_blank" rel="noopener noreferrer">
-                                {item.name}
-                            </a>
+                            <ExternalBlueprintImage url={image_url} width={30} /> {name}
                         </DataTable.Data>
-                        <DataTable.Data>{item.description}</DataTable.Data>
-                        <DataTable.Data>{item.created_at}</DataTable.Data>
-                        <DataTable.Data>{item.updated_at}</DataTable.Data>
+                        <DataTable.Data>{description}</DataTable.Data>
+                        <DataTable.Data>{created_at}</DataTable.Data>
+                        <DataTable.Data>{updated_at}</DataTable.Data>
                         <DataTable.Data className="center aligned rowActions">
+                            <StyledLink href={html_url} target="_blank" rel="noopener noreferrer">
+                                <Icon name="github" bordered />
+                            </StyledLink>
```

Comment:
What about having something like:
```suggestion
                            <Icon link name="github" bordered onClick={() => Stage.Utils.Url.redirectToPage(html_url)} />
```
here? That way we wouldn't need `StyledLink`.

Reply:
adjusted in newer commit: https://github.com/cloudify-cosmo/cloudify-stage/pull/2187/commits/37c8813cd00b412131a5bb267d49e6d4be0e65b5